src/settings/components/account-settings/settings-sent-messages.tsx

```diff
@@ -64,42 +67,54 @@ const SettingsSentMessages = ({
 		[availableEmailAddresses]
 	);
 	const [fromAddress, setFromAddress] = useState(
-		() => find(fromAddressArray, (item) => item.value === identity.fromAddress) ?? blankItem
+		() =>
+			find(fromAddressArray, (item) => item.value === identityAttrs.zimbraPrefFromAddress) ??
+			blankItem
 	);
 
 	useEffect(() => {
-		setReplyToEnabledValue(identity.replyToEnabled === 'TRUE');
-	}, [identity.replyToEnabled]);
+		setReplyToEnabledValue(identityAttrs.zimbraPrefReplyToEnabled === 'TRUE');
+	}, [identityAttrs.zimbraPrefReplyToEnabled]);
 	useEffect(() => {
-		setFromDisplayValue(identity.fromDisplay || '');
-	}, [identity.fromDisplay]);
+		setFromDisplayValue(identityAttrs.zimbraPrefFromDisplay || '');
+	}, [identityAttrs.zimbraPrefFromDisplay]);
 	useEffect(() => {
-		const k = find(fromAddressArray, (item) => item.value === identity.fromAddress) ?? blankItem;
-		setFromAddress(k);
-	}, [fromAddressArray, identity.fromAddress]);
+		setFromAddress(
+			find(fromAddressArray, (item) => item.value === identityAttrs.zimbraPrefFromAddress) ??
+				blankItem
+		);
+	}, [fromAddressArray, identityAttrs.zimbraPrefFromAddress]);
 	useEffect(() => {
-		setReplyToDisplay(identity?.replyToDisplay === undefined ? '' : identity?.replyToDisplay);
-	}, [identity?.replyToDisplay]);
+		setReplyToDisplay(
+			identityAttrs.zimbraPrefReplyToDisplay === undefined
+				? ''
+				: identityAttrs.zimbraPrefReplyToDisplay
+		);
```

Comment:
This one can be shortened to
```suggestion
		setReplyToDisplay(identityAttrs.zimbraPrefReplyToDisplay ?? '');
```